src/main/java/com/mps/deepviolet/suite/json/CipherSuite.java

```diff
@@ -3,8 +3,43 @@
 public class CipherSuite {
 
 	private String name;
+	private String keyExchange;
+	private String authentication;
+	private String algorythm;
```

Comment:
I think you want `algorithm` here (and throughout).

Reply:
@javabeanz :  Yes, I'm with @kingthorin if you change methods to algorithm I will go ahead and merge the changes.  Sorry, I know I have been sitting on this for awhile.

Reply:
I can see that the new OWASP board members election is in progress. Good luck Milton !

Reply:
It's all up to OWASP members and who they want for their board at this point.  Go ahead and change the  "algorythm" references to "algorithm".  I feel bad about dragging my feet on this for so long.

Reply:
@javabeanz any news on addressing that issue?